src/NuGetPackageConfigConverter/Commands/PackageSystemUpdateCommand.cs

```diff
@@ -52,6 +53,10 @@ private void MenuItemBeforeQueryStatus(object sender, EventArgs e)
 
         private async void MenuItemCallback(object sender, EventArgs e)
         {
+#if LAUNCHDEBUGGER
```

Comment:
I'd rather not have this in here

Reply:
I can remove this, but I find this easier than using the experimental hive.  However, I tried to use the debugger to debug the vsix, alas it didn't work.  The 2017 debugger crashes when you try to step into